app/static/js/volunteerProfile.js

```diff
@@ -109,3 +109,15 @@ $(document).ready(function(){
   });
 
 });
+
+function updateManagers(el, volunteerusername, action ){// retrieve the data of the studnet staff and program id if the boxes are checked or not
```

Comment:
`volunteerusername` should be `volunteer_username` to be consistent with your other variables